internal/server/email/sendgrid.go

```diff
@@ -65,6 +68,11 @@ func SendTemplate(name, address string, template EmailTemplate, data any) error
 		return ErrNotConfigured
 	}
 
+	if name == "" {
+		// until we have real user names
+		name = BuildNameFromEmail(address)
```

Comment:
I don't think we need to over rotate on this. Instead, just set the name to the email until we have actual user names

Reply:
the experience is pretty nice vs email addresses. I created this function for another spot that uses the name in the mail body and looked a lot better than "joe.smith@example.com invited you to ...."